deploy/agent/config_map.yaml

```diff
@@ -45,7 +48,7 @@ data:
           "secondsBetweenRequests": 5
         },
         "lfc": {
-          "port": 9399,
+          "port": 9499,
```

Comment:
why?

Reply:
9399 is the "public" (stored in ext-metrics-*) metrics. 9499 is the metrics exporter just for autoscaling. With neondatabase/neon#8298, the latter actually is higher resolution, because we don't need to worry about storage.